drivers/frequency/ad9528/ad9528.c

```diff
@@ -380,12 +380,13 @@ int32_t ad9528_setup(struct ad9528_dev **device,
 		return ret;
 
 	/* GPIO */
-	if(init_param.hw_reset_en) {
-		ret = gpio_get(&dev->gpio_resetb, &init_param.gpio_resetb);
-		if (ret < 0)
-			return ret;
+	ret = gpio_get_optional(&dev->gpio_resetb, init_param.gpio_resetb);
+	if (ret < 0)
```

Comment:
Should the driver use the error.h definitions?

Reply:
Anyhow, I guess this is the subject of a different commit/pull-request.